packages/core-modules/contracts/submodules/election/ElectionBase.sol

```diff
@@ -49,9 +49,9 @@ contract ElectionBase is ElectionStorage, InitializableMixin {
     event EpochStarted(uint epochIndex);
     event CouncilTokenCreated(address proxy, address implementation);
     event CouncilTokenUpgraded(address newImplementation);
-    event CouncilMemberAdded(address member);
-    event CouncilMemberRemoved(address member);
-    event CouncilMembersDismissed(address[] members);
+    event CouncilMemberAdded(address indexed member, uint indexed epochIndex);
+    event CouncilMemberRemoved(address indexed member);
```

Comment:
@ajsantander you forgot it here ser

Reply:
I suppose you mean that this event needs to also track `epochIndex`. On it.

Reply:
Done!